facebook-commerce.php

```diff
@@ -4240,49 +4240,57 @@ function fb_duplicate_product_reset_meta( $to_delete ) {
 
 	/**
 	 * Helper function to update FB visibility.
+	 *
+	 * @param int $product_id product ID
+	 * @param string $visibility visibility
 	 */
-	function update_fb_visibility( $wp_id, $visibility ) {
+	function update_fb_visibility( $product_id, $visibility ) {
 
 		// bail if the plugin is not configured properly
 		if ( ! $this->is_configured() || ! $this->get_product_catalog_id() ) {
 			return;
 		}
 
-		$woo_product = new WC_Facebook_Product( $wp_id );
+		$woo_product = new \WC_Facebook_Product( $product_id );
```

Comment:
can we drop instantiating this object? doesn't seem to help us in any way in the context of this method

I'd just replace it with `wc_get_product()`

Reply:
similar thing in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1314

Reply:
I'm okay with replacing with `wc_get_product` and just checking `$woo_product instanceof \WC_Product`